examples/custom_labels.py

```diff
@@ -6,17 +6,13 @@
 epsagon.init(
     token='my-secret-token',
     app_name='my-app-name',
-    metadata_only=False
+    metadata_only=False  # Optional
 )
 
 
 @epsagon.lambda_wrapper
-def handle():
+def handle(event, context):
     epsagon.label('label', 'something_to_filter_afterwards')
     epsagon.label('number_of_records_parsed_successfully', 42)
 
     return 'It worked!'
-
-
-if __name__ == '__main__':
```

Comment:
Why not leave the main for this example, to allow running it using `python custom_labels.py` ?

Reply:
Because it's a lambda handler in this example, and mostly it doesn't make sense to run it like that (so I think it's just confusing).